tests/pytests/test_option.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+import pytest
+
+import psi4
+
+pytestmark = pytest.mark.quick
+
+
+@pytest.mark.smoke
+def test_psi4_basic():
```

Comment:
```suggestion
def test_spacious_option():
```

this is fine -- I just need to trigger CI, since Azure didn't pick up your last commit